pkg/npm/publish.go

```diff
@@ -176,7 +180,7 @@ func (exec *Execute) publish(packageJSON, registry, username, password string, p
 			}
 		}
 
-		err = execRunner.RunExecutable("npm", "publish", "--tarball", tarballFilePath, "--userconfig", filepath.Join(tmpDirectory, ".piperNpmrc"), "--registry", registry)
+		err = execRunner.RunExecutable("npm", "publish", "--tarball", tarballFilePath, "--userconfig", ".piperNpmrc", "--registry", registry)
```

Comment:
```suggestion
		err = execRunner.RunExecutable("npm", "publish", "--tarball", tarballFilePath, "--userconfig", npmrc.filepath, "--registry", registry)
```
Small thing, but this makes it consistent throughout the code, and reduces hardcoding. Previously, `.piperNpmrc` was created in a temp dir inside of this function, but now the filename directly comes from `NewNPMRC()`.

Reply:
`npmrc.filepath` will the full qualified path , but since we have already chnaged directory earlier to run the npm pack, this will lead to a file not found error, hence have hard code `.piperNpmrc` since we know that the file will be there in the current directory